lib/core/domain/event/input/event_input/event_input.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'event_input.freezed.dart';
+part 'event_input.g.dart';
+
+@freezed
+class EventInput with _$EventInput {
+  @JsonSerializable(includeIfNull: false)
+  factory EventInput({
+    required String title,
+    required String description,
+    required bool private,
+    required bool verify,
+    required String start,
```

Comment:
for DateTime value, let define it DateTime, freeze library will auto convert to String when to json serialize